portfolio.css

```diff
@@ -2108,4 +1064,805 @@ input[type=email] {
+  }
+
+  .header {
+    display: none;
+  }
+
+  .desktop-headline {
+    display: none;
+  }
+
+  .desktop-social-media-icons {
+    display: none;
+  }
+
+  .desktop-work {
+    display: none;
+  }
+
+  .desktop-about-me {
+    display: none;
+  }
+
+  .desktop-form {
+    display: none;
+  }
+
+  .rect-42-image {
+    display: none;
+  }
+
+  footer {
+    display: none;
+  }
+}
+
+/* Media queries for the desktop version breakthrough */
+
+@media only screen and (min-width: 367px) and (max-height: 3846px) {
+  html,
+  body {
+    position: relative;
+    margin: 0;
+    padding: 0;
+    max-width: 2063px;
+    overflow-x: hidden;
+    overflow-y: scroll;
+  }
+
+  .cellular {
+    display: none;
+  }
+
+  .headline {
+    display: none;
+  }
+
+  .work {
+    display: none;
+  }
+
+  .about-me {
+    display: none;
+  }
+
+  .contact-form {
+    display: none;
+  }
+
+  .desktop-background-images {
+    position: absolute;
+    width: 2063px;
+    height: 1434.3px;
+    background: url('./images/desktop_version/frame_headline/Group-35.png') 40% 5% no-repeat, url('./images/desktop_version/frame_headline/Frame.png') 74% 1% no-repeat, url('./images/desktop_version/frame_headline/Group-38.png') 36% 8% no-repeat, url('./images/desktop_version/frame_headline/Group2.png') 1% 21% no-repeat, url('./images/desktop_version/frame_headline/Frame-big.png') 100% 17% no-repeat, url('./images/desktop_version/frame_headline/Group-37.png') 18% 58% no-repeat, url('./images/desktop_version/frame_headline/Frame-middle.png') 40% 50% no-repeat, url('./images/desktop_version/frame_headline/Frame-right.png') 75% 55% no-repeat;
+  }
+
+  .header {
+    width: 2063px;
+    height: 72px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .menu {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 8px;
+    padding: 0;
+  }
+
+  .menu-items1 {
+    list-style-type: none;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 87px;
+    height: 40px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    text-align: center;
+    color: #344563;
+  }
+
+  .menu-items2 {
+    list-style-type: none;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 69px;
+    height: 40px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    text-align: center;
+    color: #344563;
+  }
+
+  .menu-items3 {
+    list-style-type: none;
+    padding: 10px 12px;
+    gap: 10px;
+    width: 83px;
+    height: 40px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    text-align: center;
+    color: #344563;
+  }
+
+  .desktop-headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 160px 0;
+    gap: 24px;
+    width: 2063px;
```

Comment:
- [ ]  Kindly note that **_`Headline section`_** for the desktop view is currently not centered. This is attributed to the width you specified for the section ( width: 2063px; ). Kindly remove the width and the section will be centralized 👍🏽 .
![image](https://user-images.githubusercontent.com/63473584/187044653-b521f492-ab51-498e-a016-cc29803dad19.png)